components/landing/highlights.json

```diff
@@ -163,7 +163,7 @@
       "title": "",
       "text": "Blocking of LGBT community and news websites",
       "report": "https://sinarproject.org/digital-rights/updates/online-lgbt-censorship-malaysia",
-      "explore": "/search?input=gaystarnews.com&probe_cc=MY",
+      "explore": "/search?domain=gaystarnews.com&probe_cc=MY",
```

Comment:
This query is missing date ranges. cc @agrabeli

Reply:
Added.